charts/dependency-track/templates/_helpers.tpl

```diff
@@ -63,6 +62,7 @@ API server selector labels
 {{ include "dependencytrack.commonSelectorLabels" . }}
 app.kubernetes.io/name: {{ printf "%s-api-server" (include "dependencytrack.name" .) }}
 app.kubernetes.io/component: api-server
+app.kubernetes.io/version: {{ (.Values.apiServer.image.tag | default .Chart.AppVersion) | quote }}
```

Comment:
What's the reason for moving these to selector labels? To my understanding, `spec.selector.matchLabels` [is immutable](https://handbook.giantswarm.io/docs/dev-and-releng/app-developer-processes/matchlabels-are-immutable/). The version is expected to change, if we add it to selector labels users will have to delete and recreate resources for every upgrade.

Reply:
You are right; I didn't mean to do that. Moved it to the correct labeling section now.